src/frontend/src/layouts/Sidebar/Sidebar.tsx

```diff
@@ -71,12 +71,19 @@ const Sidebar: React.FC<SideBarProps> = ({ open, handleDrawerClose }) => {
         <IconButton onClick={handleDrawerClose}>{theme.direction === 'rtl' ? <ChevronRight /> : <ChevronLeft />}</IconButton>
       </DrawerHeader>
       <Divider />
-      <Box overflow={'auto'} sx={{ overflowX: 'hidden' }}>
+      <Box
+        overflow={'auto'}
+        sx={{ overflowX: 'hidden' }}
+        display="flex"
+        flexDirection={'column'}
+        flex={1}
+        justifyContent={'space-between'}
+      >
         {linkItems.map((linkItem) => (
```

Comment:
all the linkItems should be under one box, so one box for all of the linkitems together